### PR TITLE
Fixup links

### PR DESCRIPTION
**Pull request by tmandry**
This is the result of running the script from #238 on all markdown files.

### CODE REVIEW

File: src/vision/submitted_stories/status_quo/barbara_bridges_sync_and_async.md [hunk 1]
```diff
@@ -325,11 +325,11 @@ There are times when converting synchronous code to async is difficult or even i
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_bridges_sync_and_async.md:335]: Should this one preserve the anchor too?

File: src/vision/submitted_stories/status_quo/barbara_builds_an_async_executor.md [hunk 2]
```diff
@@ -162,11 +162,11 @@ Barbara finished her initial implementation of the async executor. Despite there
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_builds_an_async_executor.md:172]: Here too?

File: src/vision/submitted_stories/status_quo/barbara_carefully_dismisses_embedded_future.md [hunk 3]
```diff
@@ -320,9 +320,9 @@ instead. Here are some ways in which these APIs are lighter weight than a
...
-[Niklaus]: ../characters/niklaus.md
-[Barbara]: ../characters/barbara.md
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_carefully_dismisses_embedded_future.md:328]: And here?

File: src/vision/submitted_stories/status_quo/barbara_gets_burned_by_select.md [hunk 4]
```diff
@@ -73,12 +73,12 @@ This is one key point that makes cancellation in async Rust rather different tha
...
-[YouBuy]: ../projects/YouBuy.md
\ No newline at end of file
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_gets_burned_by_select.md:83]: Anchor text here too?

File: src/vision/submitted_stories/status_quo/barbara_makes_their_first_steps_into_async.md [hunk 5]
```diff
@@ -67,11 +67,11 @@ Like the author of this story, Barbara had previous experience with Rust. Knowin
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_makes_their_first_steps_into_async.md:77]: And here?

File: src/vision/submitted_stories/status_quo/barbara_needs_async_helpers.md [hunk 6]
```diff
@@ -200,14 +200,14 @@ This particular issue impacts all users of Rust even (and sometimes especially)
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_needs_async_helpers.md:210]: And here?

File: src/vision/submitted_stories/status_quo/barbara_plays_with_async.md [hunk 7]
```diff
@@ -384,10 +384,10 @@ to resort to an internet search or asking on a rust community.
...
 * [Alan started trusting the rust compiler][trusts the rust compiler] is a similar story about a different character.
 
-[status quo stories]: ./status_quo.md
-[Barbara]: ../characters/barbara.md
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[status quo stories]: ../status_quo.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_plays_with_async.md:390]: And here?

File: src/vision/submitted_stories/status_quo/barbara_tries_async_streams.md [hunk 8]
```diff
@@ -47,11 +47,11 @@ Barbara is an experienced engineer who may come to async streams and async/await
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_tries_async_streams.md:57]: And here?

File: src/vision/submitted_stories/status_quo/barbara_tries_unix_socket.md [hunk 9]
```diff
@@ -321,11 +321,11 @@ Barbara have some experience with synchronous and high-level asynchronous rust l
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_tries_unix_socket.md:331]: And here?

File: src/vision/submitted_stories/status_quo/barbara_trims_a_stacktrace.md [hunk 10]
```diff
@@ -164,11 +164,11 @@ at lib.rs:102:13
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_trims_a_stacktrace.md:174]: And here?

File: src/vision/submitted_stories/status_quo/barbara_wants_async_insights.md [hunk 11]
```diff
@@ -67,11 +67,11 @@ Depending on what languages he was using before, [Alan] would likely have had ex
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_wants_async_insights.md:77]: And here?

File: src/vision/submitted_stories/status_quo/barbara_wants_single_threaded_optimizations.md [hunk 12]
```diff
@@ -116,11 +116,11 @@ As an experienced Rustacean, [Barbara] is more likely to be responsible for desi
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_wants_single_threaded_optimizations.md:126]: And here?

File: src/vision/submitted_stories/status_quo/barbara_wants_to_use_ghostcell.md [hunk 13]
```diff
@@ -404,11 +404,11 @@ The other characters perhaps wouldn't have heard of statically-checked
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/barbara_wants_to_use_ghostcell.md:414]: And here?

File: src/vision/submitted_stories/status_quo/grace_deploys_her_service.md [hunk 14]
```diff
@@ -1,11 +1,11 @@
...
-[Grace]: ../characters/grace.md
-[Niklaus]: ../characters/niklaus.md
-[Barbara]: ../characters/barbara.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
 
-[Grace deploys her service and is able to fix problems]: ./shiny_future.md#grace-deploys-her-service-and-is-able-to-fix-problems
+[Grace deploys her service and is able to fix problems]: ../shiny_future.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/grace_deploys_her_service.md:8]: Missing anchor here too.

File: src/vision/submitted_stories/status_quo/grace_tries_new_libraries.md [hunk 15]
```diff
@@ -119,7 +119,7 @@ From her background she has an understanding what could go wrong. So she remembe
...
 * So it probably doesn't matter that much which character is used, except that Barbara may be more likely to have seen how to solve it.
 
-[character]: ../characters.md
-[status quo stories]: ./status_quo.md
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/grace_tries_new_libraries.md:125]: Missing anchor.

File: src/vision/submitted_stories/status_quo/grace_waits_for_gdb_next.md [hunk 16]
```diff
@@ -95,11 +95,11 @@ I needed someone who, like me, would actually be tempted to use `gdb` even when
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/grace_waits_for_gdb_next.md:105]: Here too.

File: src/vision/submitted_stories/status_quo/grace_wants_a_zero_copy_api.md [hunk 17]
```diff
@@ -72,7 +72,7 @@ when trying to expose zero-copy APIs.
...
 * There is a chance Barbara would have ventured into similar problems. She would likely have had a similar experience than Grace.
 
-[character]: ../characters.md
-[status quo stories]: ./status_quo.md
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/grace_wants_a_zero_copy_api.md:78]: And here.

File: src/vision/submitted_stories/status_quo/grace_writes_a_new_fb_service.md [hunk 18]
```diff
@@ -62,11 +62,11 @@ Had I chosen Alan, a Python programmer at Facebook, there is probably a lot
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/grace_writes_a_new_fb_service.md:72]: And here.

File: src/vision/submitted_stories/status_quo/niklaus_simulates_hydrodynamics.md [hunk 19]
```diff
@@ -62,11 +62,11 @@ I chose Niklaus as the primary character in this story because this work was dri
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/niklaus_simulates_hydrodynamics.md:72]: And here.

File: src/vision/submitted_stories/status_quo/niklaus_wants_to_share_knowledge.md [hunk 20]
```diff
@@ -20,8 +20,8 @@ Niklaus was excited about [the RFC proposing that `block_on` be added to the std
 [trpl]: https://doc.rust-lang.org/stable/book/
 [people like Barbara]: https://github.com/rust-lang/wg-async-foundations/blame/5ce418ac4076850f515034010cc51b707441f695/src/vision/status_quo/barbara_makes_their_first_steps_into_async.md#L22
 [block-on-rfc]: https://github.com/rust-lang/rust/pull/65875
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/niklaus_wants_to_share_knowledge.md:24]: And here.

File: src/vision/submitted_stories/status_quo/template.md [hunk 21]
```diff
@@ -43,11 +43,11 @@ If you would like to expand on this story, or adjust the answers to the FAQ, fee
...
-[htvsq]: ../how_to_vision/status_quo.md
-[cannot be wrong]: ../how_to_vision/comment.md#comment-to-understand-or-improve-not-to-negate-or-dissuade
+[character]: ../../characters.md
+[status quo stories]: ../status_quo.md
+[Alan]: ../../characters/alan.md
+[Grace]: ../../characters/grace.md
+[Niklaus]: ../../characters/niklaus.md
+[Barbara]: ../../characters/barbara.md
+[htvsq]: ../status_quo.md
+[cannot be wrong]: ../../how_to_vision/comment.md
```
Comment (eholk) [inline src/vision/submitted_stories/status_quo/template.md:53]: And here.

**Review by eholk — APPROVED** [review]
Overall looks good, but it looks like maybe the link fixing tool doesn't preserve the `#` part of the link. We probably want to try and keep that.

**Comment by tmandry** [comment]
Ah, good catch! Will fix.

**Pull request merged**